lib/widgets/movie/movie_details.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/shared/circular_percent_indicator.dart';
+import 'package:movie_card/shared/custom_card.dart';
+
+class MovieDetails extends StatelessWidget {
+  const MovieDetails({
+    super.key,
+    required this.pad1,
+    required this.title,
+    required this.date,
+    required this.percentIndicator,
+  });
+
+  final double pad1;
+  final String title;
+  final String date;
+  final double percentIndicator;
+
+  static const double opacity_1 = 0.75;
+  static const double padCard = 8.0;
+
+  static const double lineWidthPercentIndicator = 5.0;
+
+  static const double radiusPercentIndicator = 20.0;
+  static const double textGenderWidth = 140;
+
+  static const double padTitle = 5;
+  static const double padGenres = 12;
+  static const double separation = 10;
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: Padding(
+        padding: const EdgeInsets.all(padCard),
+        child: Column(
+          mainAxisSize: MainAxisSize.max,
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          children: [
+            CustomCard(
+              padding: padTitle,
+              child: ListTile(
+                title: Text(title),
+                subtitle: Text(date),
+                textColor: Theme.of(context).colorScheme.background,
+              ),
+            ),
+            const SizedBox(
+              height: separation,
+            ),
+            CustomCard(
+              padding: padGenres,
+              child: Row(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.all(padCard),
+                    child: Padding(
+                      padding: EdgeInsets.all(pad1),
+                      child: CircularPercentIndicator(
+                        percentIndicator: percentIndicator,
+                        radiusPercentIndicator: radiusPercentIndicator,
+                      ),
+                    ),
+                  ),
+                  Center(
+                    child: SizedBox(
+                      width: textGenderWidth,
+                      child: Text(
+                        "Animation, Family, Adventure, Fantasy, Comedy",
```

Comment:
These shouldn't be neither hard-coded nor static. These'll come as a list of strings in the future